shell/src/main/java/alluxio/cli/fs/command/LeaderCommand.java

```diff
@@ -53,6 +58,16 @@ public int run(CommandLine cl) {
       try {
         InetSocketAddress address = client.get().getAddress();
         System.out.println(address.getHostName());
+
+        List<InetSocketAddress> addresses = Arrays.asList(address);
+        MasterInquireClient inquireClient = new PollingMasterInquireClient(addresses, () ->
+                new ExponentialBackoffRetry(50, 100, 2)
+        );
+        try {
+          inquireClient.getPrimaryRpcAddress();
+        } catch (UnavailableException e) {
+          System.out.println("Failed to get the leader master RPC address.");
```

Comment:
At this point, we have successfully determined which host is designated as the master, it's just that the master isn't currently serving. Can you change this to be a warning instead of a failure message?

Reply:
Like this "Warning: Failed to get the leader master RPC address."?

Reply:
Lets make it just "The leader is not currently serving requests"

Reply:
Done